src/util/codecs.js

```diff
@@ -74,7 +74,7 @@ export const codecsForPlaylist = function(master, media) {
 
   // HLS with multiple-audio tracks must always get an audio codec.
   // Put another way, there is no way to have a video-only multiple-audio HLS!
-  if (isMaat(master, media) && !codecInfo.audioCodec) {
+  if (isMaat(master, media) && !codecInfo.audio) {
```

Comment:
This is actually a line of code that was missed in the orignal pull request. `parseCodecs` returns `audio` and `video` now, not `audioCodec` and `videoCodec`.